### ISSUE TITLE
CPU环境win10系统运行finetune脚本无报错无结果问题

### ISSUE THREAD

**Issue by krodio** [PaddlePaddle/PaddleNLP#2886]
doccano标注后，win10CPU环境进行finetune步骤，运行指令： python finetune.py --train_path ./data/train.txt --dev_path ./data/dev.txt --save_dir ./checkpoint --learning_rate 1e-5 --batch_size 2 --max_seq_len 512 --num_epochs 100 --model uie-tiny --seed 1000 --logging_steps 10 --valid_steps 100 --device cpu
- 页面无报错也未能正常提示成功。
- 软件环境：PaddleNLP 2.3.1，PaddlePaddle 2.3.0，win10系统，Python3.8.13。
- 硬件环境：
                System Model: HP EliteBook 820 G3
                BIOS: Default System BIOS (type: BIOS)
                Processor: Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz (4 CPUs), ~2.4GHz
                Memory: 8192MB RAM
                Available OS Memory: 8072MB RAM
                Page File: 7860MB used, 1235MB available
                Windows Dir: C:\Windows
                DirectX Version: DirectX 12
                DX Setup Parameters: Not found
                User DPI Setting: 120 DPI (125 percent)
                System DPI Setting: 120 DPI (125 percent)

**Comment by MachineSheep** [comment]
是不是在安装uie-base？如果uie-base已经安装，麻烦贴一下输出的截图

**Comment by wawltor** [comment]
这个可能是cpu训练速度太慢导致的，你可以尝试--logging_steps 10 改成 --logging_steps 1

**Comment by krodio** [comment]
> 是不是在安装uie-base？如果uie-base已经安装，麻烦贴一下输出的截图

uie-base 跟uie-tiny都安装了

**Comment by krodio** [comment]
> 这个可能是cpu训练速度太慢导致的，你可以尝试--logging_steps 10 改成 --logging_steps 1

改了运行还是一样的情况：
![image](https://user-images.githubusercontent.com/109476398/181275738-0f2f27c7-9d9a-451a-a3b8-06d5d5e1d3aa.png)

**Comment by krodio** [comment]
> 这个可能是cpu训练速度太慢导致的，你可以尝试--logging_steps 10 改成 --logging_steps 1

anyway起码报下错嘛

**Comment by wawltor** [comment]
是否方便提供一下你的测试数据了，我们来测试一下

**Comment by krodio** [comment]
> 是否方便提供一下你的测试数据了，我们来测试一下
麻烦帮瞄下，谢谢
[43778301-f58f-47ee-a16f-c9249d3ef8d0.zip](https://github.com/PaddlePaddle/PaddleNLP/files/9226653/43778301-f58f-47ee-a16f-c9249d3ef8d0.zip)
！

**Comment by krodio** [comment]
用到aistudio免费环境来做也不行，一样没有报错没有日志就结束了
[dtdata.zip](https://github.com/PaddlePaddle/PaddleNLP/files/9276161/dtdata.zip)

**Comment by krodio** [comment]
是自己低级错误把finetune代码搞错了，关了关了，占用大家时间不好意思
